phpBB/install/convert/convertor.php

```diff
@@ -224,13 +224,7 @@ function convert_data($converter)
 			trigger_error('NO_SUCH_SEARCH_MODULE');
 		}
 
-		$error = false;
-		$convert->fulltext_search = new $search_type($error, $phpbb_root_path, $phpEx, $auth, $config, $db, $user, $phpbb_dispatcher);
-
-		if ($error)
-		{
-			trigger_error($error);
-		}
+		$convert->fulltext_search = new $search_type($config, $db, $phpbb_dispatcher, $user, $phpbb_root_path, $phpEx);
```

Comment:
This isn't working, classes have different arguments in the constructors now, can I just comment this? It doesn't do anything.